src/django/api/helpers.py

```diff
@@ -61,3 +62,23 @@ def get_list_contributor_field_values(item, fields):
             index = list_fields.index(f['column_name'])
             f['value'] = data_values[index]
     return fields
+
+
```

Comment:
This is mentioned in the commit message, but in case that is missed, this was moved here to avoid a circular import in when importing `api.facility_type_processing_type` variables into `api.models`. `api.models` is imported into `api.matching` (where `clean` was originally defined). Then `api.facility_type_processing_type` (which uses the `clean` function) was imported into `api.models` to get access to the facility/processing type taxonomy. This caused a circular import error.